### DISCUSSION TITLE
Bosch BTH-RA Thermostat 2 - Permanent Value for Pi Heating Demand

### DISCUSSION THREAD

**Discussion by johnnypr0x** [Koenkk/zigbee2mqtt#19684] [Devices]
Hello,

I was wondering if you could set a permanent value for heating demand? 

Cause right now the thermostat will try to regulate the heating demand to match the demandend temperature. This means if you set a value for heating demand it will change again after approximately 15minutes. Can you somehow disable this automatism?

I made a simple automation in Home Assistant to change the value everytime it updates itself but Im looking for a permanent solution. 

![image](https://github.com/Koenkk/zigbee2mqtt/assets/48360938/2f1bcc4c-2f86-4348-95b6-02769a5fc772)

I appreciate any help.
Michael

**Comment by Starfirepl** [comment]
I have the same problem :( can you explain how you automated it to avoid this problem? I need to turn on the thermostat to 100% for a certain period of time.

**Comment by DrJohnsn** [comment]
Same here. Is there any update about this? 
Any chance to stop the own algorithm ?

**Comment by titro** [comment]
Same requirement here...

This is my temperature

![image](https://github.com/user-attachments/assets/ea1b1612-b2b1-4fe4-ab32-fec5da845eea)


Would be nice to set a static heating demand. Thermostats lost 5-8 % Battery lifetime during the last 4-6 weeks during the permanent adoption of the valve.
